core/services/wifi/frontend/index.html

```diff
@@ -0,0 +1,317 @@
+            :key="key"
+          >
+            <div class="cell-1"></div>
+            <div class="cell-7">
+              <div>{{ network.ssid }}</div>
+            </div>
+            <div class="cell-2">
+              <div class="d-flex flex-justify-center">
+                <i :class="networkProtectionClass(network)"></i>
+              </div>
+            </div>
+            <div class="cell-2">
+              <div class="d-flex flex-justify-center">
+                <i :class="signalStrengthClass(network)"></i>
+              </div>
+            </div>
+          </div>
+
+          <div  
+              v-if="connected_network"
+              class="dialog" 
+              data-role="dialog" 
+              id="disconnection_dialog">
+            <div class="dialog-title">{{ connected_network.ssid }}</div>
+            <div class="dialog-content">
+              <div
+                v-for="(value, name) in connected_network"
+                :key="name">
+                <span>{{name}}: {{ value }}</span><br />
+              </div>
+            </div>
+            <div class="dialog-actions">
+              <button class="button js-dialog-close" @click="disconnectFromNetwork">
+                Disconnect
+              </button>
+            </div>
+            <span class="button square closer js-dialog-close"></span>
+          </div>
+
+          <div class="dialog" data-role="dialog" id="connection_dialog">
+            <div class="dialog-title">{{ selected_network.ssid }}</div>
+            <div class="dialog-content">
+              <div
+                v-for="(value, name) in selected_network"
+                :key="name">
+                <span>{{name}}: {{ value }}</span><br />
+              </div>
+            </div>
+            <div class="dialog-actions">
+              <input
+                id="passwordInput"
+                type="password"
+                placeholder="Password"
+                data-role-checkbox="true"
+              />
+              <button class="button js-dialog-close" @click="connectToNetwork">
+                Connect
+              </button>
+            </div>
+            <span class="button square closer js-dialog-close"></span>
+          </div>
+
+        </div>
+      </div>
+    </div>
+  </body>
+
+  <script>
+    /*global Metro, axios, Vue*/
+
+    new Vue({
+      el: '#app',
+      data () {
+        return {
+          available_networks: [],
+          Metro: null,
+          connected_network: null,
+          selected_network: {},
+          alert_box: null,
+          wpa_state: 'starting',
+        }
+      },
+      mounted () {
+        this.API_URL = '/v1.0'
+        this.Metro = Metro
+        this.Metro.init()
+        setInterval(this.updateData, 5000)
+      },
+      computed: {
+        networksToShow () {
+          let showable_networks = this.available_networks.map((network) => {
+            if (network.ssid != null && network.ssid != ''){
```

Comment:
```suggestion
            if (!!network.ssid && !!network.ssid != ''){
```

This is a bit unclear, but it'll check for null, undefined and empty string all at once.

Reply:
Didn't know about that.
In fact we can make it

Reply:
In fact `if (network.ssid)` is enough here...

Reply:
![image](https://user-images.githubusercontent.com/1215497/118821813-fa0d3080-b88d-11eb-818a-744802c6b9fd.png)